### PR TITLE
Update 2.2.0 docs for components, commands

### PR DESCRIPTION
**Pull request by maysunfaisal**
Signed-off-by: Maysun J Faisal <maysunaneek@gmail.com>

**What does this PR do / why we need it**:
Updating few of the version 2.2.0 pages for components and commands

**Which issue(s) this PR fixes**:

N/A

**PR acceptance criteria**:

- [ ] Unit Tests
- [ ] E2E Tests
- [x] Documentation

**How to test changes / Special notes to the reviewer**:
CI passes

### CODE REVIEW

File: libs/docs/src/docs/2.2.0-alpha/adding-a-command-group.md [hunk 1]
```diff
@@ -5,34 +5,16 @@ description: Adding a command group
...
-    - `debug`
-
-    - `deploy`
+2. Use the following supported group kinds: `build`, `run`, `test`,
+`debug` or `deploy`
 
-3. At most, there can only be one default command for each group kind.
-    Set the default command by specifying a `true` value for
-    `isDefault`.
+1. At most, there can only be one default command for each group kind.
```
[the remark this answers is not in the document]
Comment (maysunfaisal) [reply libs/docs/src/docs/2.2.0-alpha/adding-a-command-group.md:16]: weird, the pages https://gabriel-rh.github.io/devfile-previews/PR35/docs/2.2.0-alpha/adding-a-command-group still show the numbering is correct.

Nevertheless, I'll correct it

**Review by gabriel-rh — APPROVED** [review]
/lgtm

Only a very minor suggestion which should not block a merge

We may need to back-port these changes to 2.1.0, 2.0.0, but happy to merge this first and then worry about older versions in a separate PR

File: libs/docs/src/docs/2.2.0-alpha/adding-a-kubernetes-or-openshift-component.md [hunk 2]
```diff
@@ -49,12 +49,12 @@ You can add either a `kubernetes` or `openshift` component to a devfile.
...
+    be deployed. Specify `deployByDefault: true` if you want to apply the
     component at start up.
 
-5. Associate `kubernetes` or `openshift` components with `Apply`
+5. Associate `kubernetes` or `openshift` components with `apply`
     commands wth `deploy` command group kind. If the `kubernetes` or
-    `openshift` component uses an image built by an image component
+    `openshift` component uses an image built by an `image` component
     defined in the devfile, you can create a composite `deploy` command
     to build the image and deploy the Kubernetes or openshift component.
```
[the remark this answers is not in the document]
Comment (maysunfaisal) [reply libs/docs/src/docs/2.2.0-alpha/adding-a-kubernetes-or-openshift-component.md:59]: Changed

```
--- a/libs/docs/src/docs/2.2.0-alpha/adding-a-kubernetes-or-openshift-component.md
+++ b/libs/docs/src/docs/2.2.0-alpha/adding-a-kubernetes-or-openshift-component.md
@@ -56,5 +56,5 @@ You can add either a `kubernetes` or `openshift` component to a devfile.
     commands wth `deploy` command group kind. If the `kubernetes` or
     `openshift` component uses an image built by an `image` component
     defined in the devfile, you can create a composite `deploy` command
-    to build the image and deploy the Kubernetes or openshift component.
+    to build the image and deploy the `kubernetes` or `openshift` component.
```

**Review by gabriel-rh — APPROVED** [review]
/lgtm

**Pull request merged**